WWTMVC5/Web.config

```diff
@@ -24,50 +23,11 @@
     <builders>
       <add name="Environment" mode="Greedy" prefix="WWT_" type="Microsoft.Configuration.ConfigurationBuilders.EnvironmentConfigBuilder, Microsoft.Configuration.ConfigurationBuilders.Environment"/>
       <add name="Secrets" userSecretsId="wwt_config" mode="Greedy" type="Microsoft.Configuration.ConfigurationBuilders.UserSecretsConfigBuilder, Microsoft.Configuration.ConfigurationBuilders.UserSecrets, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"/>
+      <add name="ServerConfig" userSecretsFile="C:\wwt\wwt-config.xml" mode="Greedy" type="Microsoft.Configuration.ConfigurationBuilders.UserSecretsConfigBuilder, Microsoft.Configuration.ConfigurationBuilders.UserSecrets, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"/>
```

Comment:
I'm not familiar with the tools being used here, but is this a thing where we could avoid hardcoding an absolute path? E.g. would it work to use a relative path, and add the filename to the `.gitignore` to avoid committing secrets?

Reply:
Yes, you can use either. However, this is so we can have a file on the server not part of the deployment that has the secrets. Once we identify all the secrets that file (and this entry) will be blown away so we can just use keyvault.